contracts/VoucherKernel.sol

```diff
@@ -7,10 +7,9 @@ import "@openzeppelin/contracts/utils/Address.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/Pausable.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
-import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
-import "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
 import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
-import "./interfaces/IERC1155ERC721.sol";
```

Comment:
`./interfaces/IERC1155ERC721.sol` this actual file is not deleted.

Reply:
Deleted. Will be included in next commit